src/doc/reference.md

```diff
@@ -3064,6 +3064,17 @@ of a condition expression it expects a refutable let statement. If the value of
 expression on the right hand side of the let statement matches the pattern, the corresponding
 block will execute, otherwise flow proceeds to the first `else` block that follows.
 
+```
```

Comment:
Does not execute simply means it will skip over the if block. It will still compile. If that's hard to understand I should probably rephrase it, what do you think?